README.md

```diff
@@ -55,34 +67,27 @@ cd notebooks
 ```
 
 ## Run Planet Notebooks in Docker
-Planet Notebooks relies on a complex stack of technologies that are not always easy to install and properly 
+Planet Notebooks rely on a complex stack of technologies that are not always easy to install and properly 
 configure. To ease this complexity we provide a docker container for running the notebook on docker compatible 
 systems. To install docker on your system please see docker's [documentation](https://docs.docker.com/engine/installation/)
 for your operating system.
 
 ### Download prebuilt Docker image (recommended)
-The Docker image for these notebooks is hosted in the [planetlabs/notebooks](https://hub.docker.com/r/planetlabs/notebooks)
-repo on DockerHub. To download and prepare the image for use, run:
+The Docker image for these notebooks is hosted in the [planetlabs/notebooks](https://hub.docker.com/r/planetlabs/notebooks) repo on DockerHub. To download and prepare the image for use, run:
+
 ```bash
+cd notebooks
 docker pull planetlabs/notebooks
 docker tag planetlabs/notebooks planet-notebooks
 ```
 
-### Build the Docker image
-Alternatively, you can build the docker image locally. Note, this only has to be done the first time you use it.
-After checking out this repository, you run:
-
-```bash
-cd planet-notebook-docker
-docker build --rm -t planet-notebooks .
-cd ..
-```
+If you get errors while running the Docker commands above, you might have to add `sudo` at the beginning, such as `sudo docker pull planetlabs/notebooks`.
 
-This will build and install the Docker image on your system, making it available to run. This may take some 
-time (between 10 and 20 minutes) depending on your network connection.
+If you want to re-build the Docker image yourself (not recommended), this is documented below in the "Appendix: Build the Docker image" section.
```

Comment:
I wouldn't go so far as to say "not recommended" as this is a perfectly valid avenue for e.g. developers who are adding dependencies

Reply:
Sounds good; I removed the "not recommended" string, then down in the appendix added: "This is useful if you are a developer adding dependencies or a new Jupyter notebook to this repo, for example."